js/services/scrobbler.js

```diff
@@ -6,13 +6,33 @@
 
       service.getSongInformation = function () {
         var deferred = $q.defer();
+        var songInformation = {};
         if (config.lastfm.user.length) {
```

Comment:
Actually: no. Your way would always be true, since the example config already contains the config.lastfm and config.lastfm.user. My way checks if the user has set a key.